spec/features/dependant_details_page_spec.rb

```diff
@@ -4,7 +4,7 @@
 INVALID_INTEGER_VALUES = ["two", "2 00", "2.37"].freeze
 
 RSpec.describe "Dependant details page" do
-  let(:dependant_details_header) { I18n.t("estimate_flow.dependant_details.legend") }
+  let(:dependant_details_header) { I18n.t("estimate_flow.partner_dependant_details.legend") }
```

Comment:
This felt like a strangely-worded change. Would it be better to use 'combined_...' rather than 'partner_...' in these situations?